docs/detections/api/rules/rules-api-create.asciidoc

```diff
@@ -969,6 +995,29 @@ POST api/detection_engine/rules
 `200`::
     Indicates a successful call.
 
+*Example 7*
+
+ES|QL rule that creates alerts from events that match an Excel parent process:
+
+[source,json]
+--------------------------------------------------
+POST api/detection_engine/rules
+{
+   "type":"esql",
+   "language":"esql",
+   "query":"from auditbeat-8.10.2 [metadata _id] | where process.parent.name == \"EXCEL.EXE\"",
```

Comment:
I thought all three metadata properties needed to be added to ensure alerts were de-duped, though maybe I'm misunderstanding that? Do users only need to specify the `_id` property to enable alert de-duping?

```suggestion
   "query":"from auditbeat-8.10.2 [metadata _id, _version, _index] | where process.parent.name == \"EXCEL.EXE\"",
```